### DISCUSSION TITLE
SharePoint list of members

### DISCUSSION THREAD

**Discussion by xela4n6** [O365/python-o365#904] [Q&A]
High,

thanks for this very useful library.
Any way of getting the members with access to a SharePoint Site? Maybe using kargs?

**Comment by alejcas** [comment]
Have you check the sharepoint code?

if not there check Ms graph docs and see if it's possible there.

If it's possile there it should be possible to implement that into O365

**Reply by xela4n6** [reply]
Hi,
I did check the sharepoint code. In fact, MS Graph API does not offer an endpoint to access SharePoint site groups. There is a workaround option using a hidden list "User Information List" that stores users (including groups) information about the site.
"https://graph.microsoft.com/v1.0/sites/{siteid}/lists/User Information List/items?expand=fields"

Once I have the group of members of a SP site, I would be able to list the users of that group.
Is it possible to extract the "User Information List" using Site class?
Thank you

**Reply by alejcas** [reply]
It should be possible but it's not implemented.
